language-server/src/dotty/tools/languageserver/DottyLanguageServer.scala

```diff
@@ -930,23 +935,25 @@ object DottyLanguageServer {
 
   /** Convert `signature` to a `SignatureInformation` */
   def signatureToSignatureInformation(signature: Signatures.Signature): lsp4j.SignatureInformation = {
-    val tparams = signature.tparams.map(Signatures.Param("", _))
     val paramInfoss =
-      (tparams ::: signature.paramss.flatten).map(paramToParameterInformation)
+      (signature.paramss.flatten).map(paramToParameterInformation)
     val paramLists =
-      if signature.paramss.forall(_.isEmpty) && tparams.nonEmpty then
-        ""
-      else
-        signature.paramss
-          .map { paramList =>
-            val labels = paramList.map(_.show)
-            val prefix = if paramList.exists(_.isImplicit) then "using " else ""
-            labels.mkString(prefix, ", ", "")
-          }
-          .mkString("(", ")(", ")")
-    val tparamsLabel = if (signature.tparams.isEmpty) "" else signature.tparams.mkString("[", ", ", "]")
+      signature.paramss
```

Comment:
I assume this whole duplication is because there is no good place for it, such that it could be used by both modules.

Reply:
Dotty language server should actually be removed from the repository as it is no longer necessary. It is not so easy tho as the test coverage is different. This change was only made to make the tests pass